docs/stylesheets/brand.css

```
@@ -1,6 +1,10 @@
-/* spectacles brand color overrides for Material for MkDocs.
+/* spectacles brand overrides for Material for MkDocs.
    Palette values from docs/assets/BRAND.md. */
 
+/* Fraunces (display face) for headings and the wordmark. The body and code
+   faces load from theme.font in mkdocs.yml. */
+@import url("https://fonts.googleapis.com/css2?family=Fraunces:opsz,wght@9..144,400..600&display=swap");
```

Stylelint (17.11.0)

[error] Expected "url("https://fonts.googleapis.com/css2?family=Fraunces:opsz,wght@9..144,400..600&display=swap")" to be ""https://fonts.googleapis.com/css2?family=Fraunces:opsz,wght@9..144,400..600&display=swap"" (import-notation)

(import-notation)